### ISSUE TITLE
!fatal, called at src\Database\Design\Ampersand\ADL1\Rule.hs:52:22

### ISSUE THREAD

**Issue by hanjoosten** [AmpersandTarski/Ampersand#354]
Consider
```
CONTEXT Issue IN ENGLISH
  r :: A*B [PROP]
ENDCONTEXT
```
This triggers the following fatal:
```
ampersand.exe: !fatal, called at src\Database\Design\Ampersand\ADL1\Rule.hs:52:22 in main:Database.Design.Ampersand.ADL1.Rule
             Ampersand v3.5.0[branch_#228:a2c7569*]
             error nr: 70
  Illegal property of an endo relation "r"
```
This also goes for other properties that can only be defined on endo relations. a test must be implemented to detect properties like PROP, TRN,ASY,SYM,RFX,IRF that can only be used on relations where source rel == target rel

**Comment by stefjoosten** [comment]
Remarkable! I would have expected this to be tested already in Travis. Apparently I'm too optimistic about the quality of our testing.

**Comment by hanjoosten** [comment]
Well, the issue is here that even if it was tested by Travis, the test would pass. Travis does no testing on the quality of the error message. Currently, we get a fatal, which is wrong. We need a user error.

**Comment by hanjoosten** [comment]
Fixed as a result of the fix of issue #383
